### DISCUSSION TITLE
Ayuda para rehacer la dapp en github y desplegar con vercel

### DISCUSSION THREAD

**Discussion by Alien69Flow** [community/community#142145] [New to GitHub]
Ayuda para rehacer la dapp de AlienFlowSpace DAO (DeFi, DeSci, GameFi, IPFS, QFS, ReFi, RWA, SocialFi, TradFi) en github y desplegar con vercel, la primera fase solamente es recuperar y rehacer a nivel visual y funcionalidades esenciales, Gracias!

**Comment by ghostinhershell** [comment]
Hi @AitorAilen, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
